scrapli_community/edgecore/ecs/_async.py

```diff
@@ -0,0 +1,42 @@
+"""scrapli_community.edgecore.ecs._async"""
+from scrapli.driver import AsyncNetworkDriver
+
+
+async def default_async_on_open(conn: AsyncNetworkDriver) -> None:
+    """
+    Async edgecore_ecs default on_open callable
+
+    Args:
+        conn: NetworkDriver object
+
+    Returns:
+        N/A  # noqa: DAR202
+
+    Raises:
+        N/A
+    """
+    await conn.acquire_priv(desired_priv=conn.default_desired_privilege_level)
+
+    await conn.send_command(command="terminal length 0")
+    await conn.send_command(command="terminal width 300")
```

Comment:
guessing this is the max value? or is "0" for "unlimited" or anything an option? almost certainly thats fine just curious!

Reply:
Nope, unfortunately it seems like setting the width to 0 for "unlimited" is not an option on these switches:

```
HOSTNAME-0#terminal width ?
  <80-300>  Number of character columns displayed on the terminal.
```

Reply:
Cool, yeah thats probably no big deal anyway!